.changeset/hot-insects-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+feat: add service bindings support
```

Comment:
Let's add a description of what service bindings are, and a short example of what a service configuration in wrangler.toml would look like.